### PR TITLE
fix the  accept encoding  

### PR DESCRIPTION
**Pull request by JessonChan**
all the file has been `go test`

### CODE REVIEW

File: context/acceptencoder.go [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+	encoderMap = map[string]acceptEncoder{ // all the other compress methods will ignore
+		"gzip":     gzipCompressEncoder,
+		"deflate":  deflateCompressEncoder,
+		"*":        gzipCompressEncoder, // * means any compress will accept,we prefer gzip
+		"identity": noneCompressEncoder, // identity means none-compress
+	}
+)
+
+// WriteFile reads from file and writes to writer by the specific encoding(gzip/deflate)
+
```
Comment (astaxie) [inline context/acceptencoder.go]: 这行不应该是空行吧，下面的也是

Comment (JessonChan) [inline context/acceptencoder.go]: updated

**Comment by astaxie** [comment]
@JessonChan conflicted now

**Comment by JessonChan** [comment]
@astaxie fixed

**Pull request merged**